sites/platform/src/administration/security/mfa.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Multi-Factor Authentication (MFA)
+weight: 4
+keywords: 
+- 2fa
+- twofactor
+- two factor
+- mfa
+- multifactor authentication
+description: Enhance security within your organization by enforcing MFA over SSH and the {{% vendor/name %}} API.
+banner: 
+    type: tiered-feature
+---
+
+To enhance security, you can [enforce MFA within your organization](#within-your-organization).
+This provides an extra layer of protection for connections made through both SSH and the {{% vendor/name %}} API.
+
+When MFA is enforced, every project contributor must [enable MFA for their user account](#for-your-user-account) to run Git commands, SSH into an environment, and connect to the organization through the {{% vendor/name %}} API (for example, to trigger a Console or CLI action).
+
+## Enable MFA
+
+### Within your organization
+
+As an **admin user**, to enable MFA, open a [support ticket](/learn/overview/get-support)
+and request for MFA over SSH and API to be enforced within your organization.
+
+### For your user account
+
+As a project contributor, you **must** enable MFA for your user account so you can:
+
+- Access an organization that enforces MFA in the Console
+- Successfully connect to environments in that organization via SSH or the {{% vendor/name %}} API
+
+To enable MFA for your user account, follow these steps:
+
+{{< codetabs >}}
++++
+title=Using the CLI
```

Comment:
It can only be done in Console

(the CLI can trigger this with `platform login --method mfa` but it just sends you to Console)